CONTRIBUTING.md

```diff
@@ -1,22 +1,24 @@
 # How to contribute
 
+We welcome contributions to the Mobile Buy SDK. But before you contribute, make sure that you're familiar with the guidelines below.
+
 ## Things we will merge
 
 * Bugfixes
 * Performance improvements
-* Features which are likely to be useful to the majority of users
+* Features that are likely to be useful to the majority of users
 
 ## Things we won't merge
 
-* Code which adds no significant value to the SDK
-* Code which comes without tests
-* Code which breaks existing tests
+* Code that adds no significant value to the SDK
+* Code that comes without tests
+* Code that breaks existing tests
 
 ## Workflow
 
-* Fork the repo and branch off of `develop`
-* Create a new branch in your fork
-* If it makes sense, add tests for your code 
-* New keys should be added to `test_shop_data.json`, but don't check in their values
-* Make sure all tests pass
-* Create a pull request
+1. Fork the repo and branch off of `develop`.
+2. Create a new branch in your fork.
+3. If it makes sense to add tests for your code, then do so.
+4. Add any new keys to `test_shop_data.json`, but don't check in their values.
```

Comment:
Let's get rid of `#4`, we no longer have `test_shop_data.json`. That was left over from v2.0.